crates/bevy_ui/src/widget/text.rs

```diff
@@ -22,14 +24,28 @@ fn scale_value(value: f32, factor: f64) -> f32 {
 
 /// Defines how `min_size`, `size`, and `max_size` affects the bounds of a text
 /// block.
-pub fn text_constraint(min_size: Val, size: Val, max_size: Val, scale_factor: f64) -> f32 {
+pub fn text_constraint(
```

Comment:
This looks like something that is unit testable. I'd recommend adding a few tests.

Reply:
added some unit tests